report_thesis/src/sections/background.tex

```diff
@@ -316,7 +316,59 @@ \subsubsection{Extra Trees Regressor (ETR)}
 As a tradeoff, \gls{etr} is less interpretable than a single decision tree, as the added randomness can introduce more bias than \gls{rf}.
 However, it often achieves better generalization performance, especially in high-dimensional or noisy datasets.
 
-\subsubsection{Gradient Boosting Regressor (GBR)}
+\subsubsection{Gradient Boosting Regression (GBR)}\label{sec:gradientboost}
+In this section we introduce \gls{gbr} primarily based on \citet{James2023AnIS}.
+\gls{gbr} is an ensemble learning technique that uses a sequence of weak learners, typically decision trees, to build a strong predictive model.
```

Comment:
I think it's weird to introduce the term "weak leaner" here, and then immediately after again on line 324, where you even highlight it in italics. The same goes for the decision tree name drop, which you mention on line 330. Why even have this line?

Reply:
I agree that the line doesn't contribute much, as it only says that GBR is an ensemble technique, but then gives a characteristic of ensemble models.
It would make more sense if the sentence directly touched upon what GBR is, as opposed to other ensemble models.

Like:
```suggestion
\gls{gbr} is an ensemble learning method that builds models sequentially, each one trying to correct the errors of the previous one using gradient descent and boosting techniques.
```